doc/footer.html

```diff
@@ -17,5 +17,118 @@
 </a> $doxygenversion
 </small></address>
 <!--END !GENERATE_TREEVIEW-->
+<script type="text/javascript">
+//<![CDATA[
+/*
+  StringCompare that treats followed numbers as a single one.
+  Example: "OpenCV 9" would come first than "OpenCV 10"
+*/
+function myStringCompare(ele1, ele2) {
+    var str1 = ele1.innerHTML;
+    var str2 = ele2.innerHTML;
+    var value = -1;
+    var str1_tmp_char;
+    var str2_tmp_char;
+    var smaller = str1.length;
+    var int1;
```

Comment:
define variables in the tightest scope they are used (at the line int1 = parseInt(str1.substring(i)); basically)